src/locales/en.json

```diff
@@ -48,9 +48,11 @@
       "explanation": "That link has already expired so we sent you another email with a new link.  You should receive it within the minute.  You will not need this tab anymore."
     },
     "steps": {
+      "authenticator": "Authenticator app",
+      "backupCodes": "Backup codes",
+      "securityKeyStep": "Security key",
       "pwStep": "Password",
       "pwRecoverStep": "Password recovery",
-      "2svStep": "2-step verification",
       "completeStep": "Complete"
```

Comment:
What is the sorting significance in this list? It's not alphabetical. Is it the order of the wizard? I only ask because you put the 3 new ones at the top instead of where 2svStep was.

Reply:
yeah, I should put them where 2svStep was. Thanks